src/Microsoft.OData.Edm/Csdl/Semantics/BadElements/UnresolvedPropertyRef.cs

```diff
@@ -0,0 +1,30 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="UnresolvedProperty.cs" company="Microsoft">

```

Comment:
This should be `UnresolvedPropertyRef.cs` instead of `UnresolvedProperty.cs`
```suggestion
// <copyright file="UnresolvedPropertyRef.cs" company="Microsoft">

```